libbeat/processors/decode_duration/decode_duration_test.go

```diff
@@ -0,0 +1,75 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package decode_duration
+
+import (
+	"math"
+	"testing"
+	"time"
+
+	"github.com/elastic/beats/v7/libbeat/beat"
+	"github.com/elastic/elastic-agent-libs/mapstr"
+)
+
+func testProcessor() *decodeDuration {
+	return &decodeDuration{
+		config: decodeDurationConfig{
+			Field:  "duration",
+			Format: "",
+		},
+	}
+}
```

Comment:
You don't need this function here, you can just instantiate a `decodeDuration` whenever you need it, it's just a few lines. It will make it easier to the reader of the test to know the configuration of the `decodeDuration` processor.